packages/sanity/src/structure/i18n/resources.ts

```diff
@@ -403,6 +403,10 @@ const structureLocaleStrings = defineLocalesResources('structure', {
   /** The text for the "Open preview" action for a document */
   'production-preview.menu-item.title': 'Open preview',
 
+  /** The message that will be shown to users when the table view is not supported the list has no schema type */
+  'sheet-list.not-supported.no-schema-type':
```

Comment:
Based on the team discussion today, should this be moved to a different resource that is not structure?

Reply:
Yes, will close this PR and move it to the branch where we are adding the new file with the translations.  Thanks Binoy!